rules/windows/process_creation/win_lolbin_cscript_gathernetworkinfo.yml

```diff
@@ -0,0 +1,31 @@
+title: Gather information about a system using gatherNetworkInfo.vbs script
+id: 575dce0c-8139-4e30-9295-1ee75969f7fe
+description: Adversaries can abuse the presence of C:\Windows\System32\gatherNetworkInfo.vbs script along with cscript.exe to gather information about the target
+status: experimental
+references:
+    - https://posts.slayerlabs.com/living-off-the-land/#gathernetworkinfovbs 
+author: blueteamer8699
+date: 2022/01/03
+modified: 2021/03/01
+tags:
+    - attack.discovery
+    - attack.group_policy_discovery
+    - attack.execution
+    - attack.command_and_scripting_interpreter
+    - attack.visual_basic
+    - attack.t1059_005
+logsource:
+    category: process_creation
+    product: windows
+    service: sysmon
```

Comment:
`service: sysmon` is to remove as the rule use category.
sysmon and EventID are add when convert with the /config/generic/sysmon.yml file in the sigmac option